build-release-tools/lib/RepositoryOperator.py

```diff
@@ -326,6 +326,36 @@ def get_lastest_merge_commit_before_date(self, repo_dir, date):
             raise RuntimeError("Unable to get commit id before {date} in directory {repo_dir}"\
                   .format(date=date, repo_dir=repo_dir))
 
+    def get_latest_author_commit_before_date(self, repo_dir, date, author):
+        if repo_dir is None or not os.path.isdir(repo_dir):
+            raise RuntimeError("The repository directory is not a directory")
```

Comment:
'The repository directory is not a directory' is not clear in the error log, it could be 'the repository directory is not specified. or its format is wrong'